t/ui/18-tests-details.t

```diff
@@ -446,6 +448,8 @@ my $post
 $get         = $t->get_ok($baseurl . 'tests/99963')->status_is(200);
 @worker_text = $get->tx->res->dom->find('#assigned-worker')->map('all_text')->each;
 like($worker_text[0], qr/[ \n]*Assigned worker:[ \n]*localhost:1[ \n]*/, 'worker still displayed when job set to done');
+@scenario_description = $get->tx->res->dom->find('#scenario-description')->map('all_text')->each;
+like($scenario_description[0], qr/[ \n]*Simple kde test, before advanced_kde[ \n]*/, 'scenario description is missing');
```

Comment:
This test isn't any different compared to the one in line 162, is it? And I'm wondering about the test name 'scenario description is missing'. To me it looks that quite the opposite is checked here.

Reply:
Yes, it checks the same after some work - just to make sure that description is still displayed after workflow, but I can remove it if you think it is excessive.